tuftool/Cargo.toml

```diff
@@ -35,13 +35,13 @@ simplelog = "0.8.0"
 snafu = { version = "0.6.8", features = ["backtraces-impl-backtrace-crate"] }
 structopt = "0.3"
 tempfile = "3.1.0"
-tokio = "0.2.21"
+tokio = "0.3.3"
```

Comment:
Are we sure this works, given that Rusoto is still on tokio 0.2? (I don't know the compatibility story here.)

Reply:
It does based on testing. We can back it out if concerned though.

There is a change to the interface for tokio::Runtime::new, and the tokio 0.3 breaks up the tokio project into features in a way that it didn't before. But in our build/testing here it works.

Reply:
Upon further reflection, you're right, too risky. Backed that out, CI running...